nilearn/plotting/img_plotting.py

```diff
@@ -726,13 +728,25 @@ def plot_glass_brain(stat_map_img,
     def display_factory(display_mode):
         return functools.partial(get_projector(display_mode), alpha=alpha)
 
+    # set vmin/vmax by hand
+    stat_map_img = _utils.check_niimg(stat_map_img, ensure_3d=True)
+    stat_map_data = stat_map_img.get_data()
+    # Avoid dealing with masked_array:
+    if hasattr(stat_map_data, '_mask'):
+        stat_map_data = np.asarray(stat_map_data[
+                np.logical_not(stat_map_data._mask)])
```

Comment:
If it's just for computing the max, you don't need to unmask. If you really want to get the non-masked values, use `stat_map_data.compressed()`